ReactJS/SunburstChart/Tooltip.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Tooltip
-description: Learn how to add Tooltip to Sunburstchart .
+title: Add Tooltip to Sunburst Chart | Syncfusion
```

Comment:
Tooltip in ReactJS Sunburst Chart Control | Syncfusion

Reply:
Added